ITKVTKHelpers.cpp

```diff
@@ -314,7 +313,7 @@ void CreateScalarImageFromStructuredGridArray(vtkStructuredGrid* const structure
 
       p = value[0];
 
-      imageIterator.Set(p);
```

Comment:
why do we not need this? 'p' is just a local variable, no?

Reply:
loacal variable p is in while{};so you just in if(){set p value}else{set p value},than you juse write once  imageIterator.Set(p); can make the same thing.